File: Tanmay Newatia/Day 4/26 - Remove Duplicates from sorted array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        c=0
        for i in range(len(nums)-1):
            if nums[i]==nums[i+1]:
                nums[i]=101
            else:
                c+=1
        c+=1
        nums.sort()
        return c

File: Tanmay Newatia/Day 4/26 - Remove Duplicates from sorted array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        map<int,int> mp;
        int j=0;
        for(auto i:nums){
            if(mp[i]==0){
                nums[j]=i;
                j++;
                mp[i]++;
            }
        }
        return j;
    }
};